.circleci/config.yml

```diff
@@ -686,6 +686,7 @@ jobs:
       DEFAULT_DEX_IP: "172.18.0.2"
       TEST_UPGRADE: "1"
       USE_MULTICLUSTER_OIDC_ENV: "true"
+      TEST_OPERATORS: ""
```

Comment:
Thanks for introducing the flag for CI.
Shouldn't we add another step where we test the operators?

Reply:
Oh - I'll be setting the value here to "1" before landing so that we continue to test with operators in CI. After landing I'll then look at whether to also run a job without operator support.

Reply:
OK, so I've verified that this PR [passes CI when the operators and the k8s api proxying are disabled](https://circleci.com/gh/kubeapps/kubeapps/70321?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), and have now set the `TEST_OPERATORS` to true so that it's now testing operators (and proxying to the k8s api) again to land, until we're ready to switch CI.